tests/settings/SettingTest.py

```diff
@@ -34,7 +35,7 @@ def test_path(self):
         self.assertEqual(path(self.uut),
                          os.path.abspath(os.path.join('.', '22')))
 
-        abspath = os.path.abspath('.')
+        abspath = PathArg(os.path.abspath('.'))
```

Comment:
Better move this change to #5019 since this test is not related to a cmdline arg